test/build.info

```diff
@@ -202,3 +202,7 @@ DEPEND[memleaktest]=../libcrypto
 SOURCE[dtlsv1listentest]=dtlsv1listentest.c
 INCLUDE[dtlsv1listentest]={- rel2abs(catdir($builddir,"../include")) -} .. ../include
 DEPEND[dtlsv1listentest]=../libssl
+
+SOURCE[threadstest]=threadstest.c
+INCLUDE[threadstest]={- rel2abs(catdir($builddir,"../include")) -} ../include
```

Comment:
This needs to have ".." added in the same way as for "dtlsv1listentest" above. Without that "make test" fails to build when using the unified build scheme